bin/pyspark

```diff
@@ -123,14 +94,6 @@ if [[ -n "$SPARK_TESTING" ]]; then
   exit
 fi
 
-# If a python file is provided, directly run spark-submit.
-if [[ "$1" =~ \.py$ ]]; then
-  echo -e "\nWARNING: Running python applications through ./bin/pyspark is deprecated as of Spark 1.0." 1>&2
-  echo -e "Use ./bin/spark-submit <python file>\n" 1>&2
-  primary="$1"
-  shift
-  gatherSparkSubmitOpts "$@"
-  exec "$FWDIR"/bin/spark-submit "${SUBMISSION_OPTS[@]}" "$primary" "${APPLICATION_OPTS[@]}"
-else
-  exec "$PYSPARK_DRIVER_PYTHON" $PYSPARK_DRIVER_PYTHON_OPTS
-fi
```

Comment:
$ ./bin/pyspark ./examples/src/main/python/pi.py
    WARNING: Running python applications through 'pyspark' is deprecated as of Spark 1.0.
    Use ./bin/spark-submit <python file>

That is handled in `SparkSubmitCommandBuilder#buildPySparkShellCommand`.